### PR TITLE
added asset swapper support for celo

### PR DESCRIPTION
**Pull request by dextracker**
## Description

Adds support for Celo ChainId: 42220. 
Supports two liquidity Sources:
1. UbeSwap
2. SushiSwap

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/asset-swapper/src/utils/market_operation_utils/types.ts [hunk 1]
```diff
@@ -93,6 +93,9 @@ export enum ERC20BridgeSource {
     // Avalanche
     Pangolin = 'Pangolin',
     TraderJoe = 'TraderJoe',
+    // Celo only
+    UbeSwap = 'UbeSwap',
+    UbeSwapMoola = 'UbeSwapMoola',
```
Comment (dorothy-zbornak) [inline packages/asset-swapper/src/utils/market_operation_utils/types.ts:98]: I think we should just put these behind the same `UbeSwap` source but select the router based on the tokens being traded.

File: packages/contract-addresses/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ Edit the package's CHANGELOG.json file only.
 
 CHANGELOG
 
+## v6.8.0 - _October 20, 2021_
+
+    * Celo deployment (#348)
+
```
Comment (dorothy-zbornak) [inline packages/contract-addresses/CHANGELOG.md:11]: This file is autogenerated. Delete plz

File: packages/contract-addresses/CHANGELOG.json [hunk 3]
```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "version": "6.9.0",
+        "changes": [
+            {
+                "note": "Celo deployment",
+                "pr": 348
+            }
+        ],
+        "timestamp": 1634668033
```
Comment (dorothy-zbornak) [inline packages/contract-addresses/CHANGELOG.json:10]: this field is autogenerated
```suggestion
        ]
```

File: packages/asset-swapper/src/utils/market_operation_utils/constants.ts [hunk 4]
```diff
@@ -632,6 +657,17 @@ export const DEFAULT_INTERMEDIATE_TOKENS_BY_CHAIN_ID = valueByChainId<string[]>(
...
+        [ChainId.Celo]: [
+            CELO_TOKENS.cUSD,
+            CELO_TOKENS.cEUR,
+            CELO_TOKENS.cETH,
+            CELO_TOKENS.cBTC,
+            CELO_TOKENS.mCUSD,
+            CELO_TOKENS.mCEUR,
+            CELO_TOKENS.UBE,
+            CELO_TOKENS.WCELO,
+
```
Comment (dorothy-zbornak) [inline packages/asset-swapper/src/utils/market_operation_utils/constants.ts:669]:
```suggestion
```

File: packages/asset-swapper/CHANGELOG.json [hunk 5]
```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "version": "16.31.0",
+        "changes": [
+            {
+                "note": "Celo deployment",
+                "pr": 348
+            }
+        ],
+        "timestamp": 1634668033
```
Comment (dorothy-zbornak) [inline packages/asset-swapper/CHANGELOG.json:10]:
```suggestion
        ]
```

File: packages/asset-swapper/src/utils/market_operation_utils/constants.ts [hunk 6]
```diff
@@ -1690,6 +1732,20 @@ export const TRADER_JOE_ROUTER_BY_CHAIN_ID = valueByChainId<string>(
     NULL_ADDRESS,
 );
 
+export const UBESWAP_ROUTER_BY_CHAIN_ID = valueByChainId<string>(
+    {
+        [ChainId.Celo]: '0x7D28570135A2B1930F331c507F65039D4937f66c',
```
Comment (dorothy-zbornak) [inline packages/asset-swapper/src/utils/market_operation_utils/constants.ts:1737]: ?
```suggestion
        [ChainId.Celo]: '0xE3D8bd6Aed4F159bc8000a9cD47CffDb95F96121',
```

**Comment by dextracker** [comment]
testing if this reopens pr

**Pull request merged**